.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
      build:
           working_directory: /go/src/github.com/DataDog/opencensus-go-exporter-datadog
           docker:
-               - image: circleci/golang:1.10
+               - image: circleci/golang:1.14
```

Comment:
updated because previously was failing with

https://app.circleci.com/pipelines/github/DataDog/opencensus-go-exporter-datadog/5/workflows/dba01280-9311-49cb-aa4c-7f2a175b86cf/jobs/233/parallel-runs/0/steps/0-103

then found this:
https://github.com/stretchr/testify/issues/991